crates/valence_server/src/layer/chunk/paletted_container.rs

```diff
@@ -178,8 +175,6 @@ impl<T: Copy + Eq + Default, const LEN: usize, const HALF_LEN: usize>
                     // Bits per entry
                     (direct_bits as u8).encode(&mut writer)?;
 
-                    // Number of longs in data array.
-                    VarInt(compact_u64s_len(LEN, direct_bits) as i32).encode(&mut writer)?;
```

Comment:
the length prefixes were removed in 1.21.5
https://minecraft.wiki/w/Java_Edition_protocol/Chunk_format#Paletted_Container_structure

> As of 1.21.5, the length of the data array is no longer sent with the packet, but is instead calculated from the bits per entry and the number of entries.